src/modules/block/components/blockDetails/blockDetails.css

```diff
@@ -47,3 +47,8 @@ h1 {
 .wrapper {
   margin-bottom: 25px;
 }
+
+.content {
```

Comment:
It would be nice if we could refactor away from important and be more fine grained with how important/fine grained a rule is. For example is this specific enough?
```
.wrapper {
  margin-bottom: 25px;

  & .content {
    padding-left: 0;
    padding-right: 0;
  }
}
```